yaas/parsers/bootstrap.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import getpass
+import inspect
+
+from ..bootstrap import Bootstrap
+
+
+def command(parent_parser):
+    parser = parent_parser.add_parser(
+        'bootstrap',
+        help=inspect.getdoc(Bootstrap))
+
+    subcommands = [
+        bootstrap_agents,
+        bootstrap_show,
+        ]
+
+    subparsers = parser.add_subparsers()
+    for subcommand in subcommands:
+        subcommand(subparsers)
+
+
+def bootstrap_agents(parent_parser):
+
+    """ Install and configure ambari-agent on specified hosts. """
+
+    def agents(client, args):
+        with open(args.ssh_key, 'r') as f:
+            ssh_key = f.read()
```

Comment:
That's a file you would keep on your computer, so I could see that coming from stdin. We should be consistent--any command that has an argument `--file` should be able to read from stdin (and vice versa).